<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/icons/flash.ico</normaloff>:/res/icons/flash.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>91</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ButtonSettings">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/gear.ico</normaloff>:/res/icons/gear.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonStartStop">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/play.ico</normaloff>:/res/icons/play.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonCapture">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/camera.ico</normaloff>:/res/icons/camera.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonDraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/paintbrush2.ico</normaloff>:/res/icons/paintbrush2.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonDemo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/hourglass.ico</normaloff>:/res/icons/hourglass.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonLoad">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/upload.ico</normaloff>:/res/icons/upload.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonQuit">
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/power.ico</normaloff>:/res/icons/power.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ImageView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>800</width>
     <height>681</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(163, 163, 163);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>261</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/res/icons/robowizard__logo.ico</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonSettings</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonStartStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonStartStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCapture</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonCaptureClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonDraw</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonDrawClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonLoadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonDemo</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonDemoClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ButtonSettingsClicked()</slot>
  <slot>ButtonDrawClicked()</slot>
  <slot>ButtonCaptureClicked()</slot>
  <slot>ButtonStartStopClicked()</slot>
  <slot>ButtonLoadClicked()</slot>
  <slot>ButtonDemoClicked()</slot>
 </slots>
</ui>
